tgui/packages/tgui/interfaces/GreyscaleModifyMenu.tsx

```diff
@@ -107,27 +109,29 @@ const ColorDisplay = (props, context) => {
 const PreviewCompassSelect = (props, context) => {
   const { act, data } = useBackend<GreyscaleMenuData>(context);
   return (
-    <Section>
-      <Flex mx="25%" fluid>
-        <SingleDirection dir={Direction.NorthWest} />
-        <SingleDirection dir={Direction.North} />
-        <SingleDirection dir={Direction.NorthEast} />
-      </Flex>
-      <Flex mx="25%">
-        <SingleDirection dir={Direction.West} />
-        <Flex.Item grow={1} basis={0}>
-          <Button lineHeight={3} m={-0.2} fluid>
-            <Icon name="arrows-alt" size={1.5} m="20%" />
-          </Button>
-        </Flex.Item>
-        <SingleDirection dir={Direction.East} />
-      </Flex>
-      <Flex mx="25%">
-        <SingleDirection dir={Direction.SouthWest} />
-        <SingleDirection dir={Direction.South} />
-        <SingleDirection dir={Direction.SouthEast} />
-      </Flex>
-    </Section>
+    <Box grow>
+      <Stack vertical>
+        <Flex grow fluid>
```

Comment:
> `<Flex grow>`

Flex doesn't work like that, you know.

Reply:
Ah yeah I forgot to come back here and clean stuff up after getting it to finally work. Was having issues when dev server just suddenly stopped updating uis and no changes were taking effect.